cpp/src/slice2cs/Gen.cpp

```diff
@@ -713,7 +749,7 @@ Slice::CsVisitor::writeDataMemberInitializers(const DataMemberList& dataMembers)
 void
 Slice::CsVisitor::writeDocComment(const ContainedPtr& p, const string& generatedType, const string& notes)
 {
-    optional<DocComment> comment = DocComment::parseFrom(p, csLinkFormatter, true, true);
```

Comment:
Now that I merged in my fix for doc-comment tags getting stripped out, you should add these bools back.

We still need them to escape comments like `returns true if a < b`, which would be picked up as invalid XML by docfx.